### PR TITLE
eth/*: remove redundant parentheses

### PR DESCRIPTION
**Pull request by Matthalp**

### CODE REVIEW

File: eth/handler.go [hunk 1]
```diff
@@ -392,7 +392,7 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 					unknown = true
 				} else {
 					query.Origin.Hash, query.Origin.Number = pm.blockchain.GetAncestor(query.Origin.Hash, query.Origin.Number, ancestor, &maxNonCanonical)
-					unknown = (query.Origin.Hash == common.Hash{})
+					unknown = query.Origin.Hash == common.Hash{}
```
Comment (karalabe) [inline eth/handler.go]: Please revert this change, it's cleaner in its original form.

Comment (Matthalp) [reply eth/handler.go]: Done.
